var/spack/repos/builtin/packages/pcre2/package.py

```diff
@@ -79,6 +89,7 @@ def cmake_args(self):
         args.append(self.define_from_variant("PCRE2_BUILD_PCRE2_16", "multibyte"))
         args.append(self.define_from_variant("PCRE2_BUILD_PCRE2_32", "multibyte"))
         args.append(self.define_from_variant("PCRE2_SUPPORT_JIT", "jit"))
+        args.append(self.define_from_variant("PCRE2_STATIC_PIC", "pic"))
```

Comment:
I believe this is a no-op when the shared build in enabled, testing on my local system I only get shared binaries.

Reply:
I think that's fine then. On linux both static and shared are built afaik